src/doc/rustc/src/platform-support.md

```diff
@@ -146,6 +146,9 @@ not available.
 
 target | std | host | notes
 -------|-----|------|-------
+=======
+`aarch64-apple-darwin` | ? |  | ARM64 macOS
```

Comment:
Misapplied conflict resolution

Reply:
Sorry ☺️ Fixed it

Reply:
It appears that your PR adds the Apple-Darwin target to this tier. It should not as it’s now in a higher tier.

Reply:
That was a remnant of the broken merge. Fixed that as well. Thanks!